models/note.js

```diff
@@ -1,8 +1,16 @@
 const mongoose = require('mongoose');
 
 const noteSchema = new mongoose.Schema({
-  tite: String,
-  description: String
+  tite: {type: String, required: true},
+  description: {type: String, required: true},
+  author: {
+    id: {
+      type: mongoose.Schema.Types.ObjectId,
```

Comment:
More ES6 shenanigans:
Under destructured import that I mentioned above you can further destructure Schema.Types for types you want. So:
```js
const { model, Schema } = require('mongoose');

const { ObjectId } = Schema.Types;
```